test/DynamoCoreTests/Nodes/DictionaryTests.cs

```diff
@@ -79,7 +79,7 @@ public void EmptyDictionaryToString()
             Assert.AreEqual(dict.ToString(), "{}");
         }
 
-        [Test,Category("FailureNET6")]
+        [Test]
```

Comment:
@aparajit-pratap was this the same test that you fixed?

Reply:
According to the acceptance criteria of the US, the tests that failed in net60 in this [PR](https://github.com/DynamoDS/Dynamo/pull/13766/files) were reviewed and when they were run it was found that they did not fail.

Reply:
same request as above, the AC should have been more specific, it did not mean to refer to ALL tests in the repo.

Reply:
Yes, I think I had already removed the failure category from the Dictionary test.